openfisca_aotearoa/tests/social_security/childcare_subsidy.yaml

```diff
@@ -20,16 +22,16 @@
       children:
         - "Child"
   output_variables:
-    family_has_resident_child_under_5_not_in_school:
+    social_security_regulation__family_has_resident_child_under_5_not_in_school:
       2018-08:
         - true  #  Whanau
-    family_has_resident_child_aged_5_who_will_be_enrolled_in_school:
+    social_security_regulation__family_has_resident_child_aged_5_who_will_be_enrolled_in_school:
```

Comment:
gosh the variables are getting long. I'm thinking we look into getting names spacing into the variables in OF core this year.